lighthouse/helpers/reports.py

```diff
@@ -100,7 +100,24 @@ def get_new_report_name_and_path() -> Tuple[str, pathlib.PurePath]:
 
     return report_name, report_path
 
+
 # Stip any leading zeros from the coordinate
 # eg. A01 => A1
 def unpad_coordinate(coordinate):
-    return re.sub(r'0(\d+)$', r'\1', coordinate) if (coordinate and isinstance(coordinate, str)) else coordinate
+    return (
+        re.sub(r"0(\d+)$", r"\1", coordinate)
+        if (coordinate and isinstance(coordinate, str))
+        else coordinate
+    )
+
+
+def delete_reports(filenames):
+    """delete reports from the standard reports folder if they exist.
+
+    Returns:
+        Nothing.
```

Comment:
Maybe the successfully deleted filenames could be returned from this method, and the request then displayed in the UI success alert

Reply:
I did think about that but wondered whether it was worth it? Kind of like saying this is what you are missing nothing you can do about it :(